### PR TITLE
Soapbox integration

### PR DESCRIPTION
**Pull request by jonnyarnold**
This makes reevoo.github.io use Project Soapbox as the comments provider, instead of Disqus.

### CODE REVIEW

File: Rakefile.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$LOAD_PATH.unshift File.join(File.dirname(__FILE__), 'lib')
+
+require 'soapbox_comment_linker'
+
+desc 'Load content_id for posts without content id, build and serve'
+namespace :jekyll do
+  task :serve do
+    SoapboxCommentLinker.link
+    sh 'jekyll serve --port 4001'
```
Comment (lpil) [inline Rakefile.rb]: Don't we want this run after the article is added, not before it?

Comment (jonnyarnold) [inline Rakefile.rb]: Eh?

The pipeline is:
- Write your article.
- Run `jekyll:serve`; it'll be given a Content ID and served up.
- Any future runs of `jekyll:serve` will not change the Content ID.

Comment (lpil) [inline Rakefile.rb]: OK, I generally run serve before I write the article. Maybe have a test that checks all the articles have content IDs and have it run on travis.

Comment (jonnyarnold) [inline Rakefile.rb]: It's in the readme :)

> Make sure your article has a Soapbox Content ID (you may need to re-run `bundle exec rake jekyll:serve` to get this!)

Comment (lpil) [inline Rakefile.rb]: We totally read the docs here

Comment (jonnyarnold) [inline Rakefile.rb]: I like how 'read' is present and past tense... leads to some great ambiguity.

**Comment by lpil** [comment]
LGTM

**Comment by jonnyarnold** [comment]
SOAPBOX IS BORN

**Comment by jonnyarnold** [comment]
...and it's already broken. FFS.

**Comment by jonnyarnold** [comment]
Nope, all good :dancer:

**Pull request merged**